internal/services/loadbalancer/backend_address_pool_address_resource.go

```diff
@@ -78,15 +80,21 @@ func (r BackendAddressPoolAddressResource) Arguments() map[string]*pluginsdk.Sch
 
 		"virtual_network_id": {
 			Type:         pluginsdk.TypeString,
-			Required:     true,
+			Optional:     true,
 			ValidateFunc: networkValidate.VirtualNetworkID,
 		},
 
 		"ip_address": {
 			Type:         pluginsdk.TypeString,
-			Required:     true,
+			Optional:     true,
 			ValidateFunc: validation.IsIPAddress,
 		},
+
+		"backend_address_ip_config_id": {
```

Comment:
shouldn't this be
```suggestion
		"backend_address_ip_configuration_id": {
```

to match the resource?